### PR TITLE
Update constants.js

### PR DESCRIPTION
**Pull request by sw1115**

### CODE REVIEW

File: app/javascript/app/data/constants.js [hunk 1]
```diff
@@ -39,18 +39,6 @@ export const GHG_CALCULATION_OPTIONS = {
   }
 };
 
-export const CHART_TYPES = {
```
Comment (Bluesmile82) [inline app/javascript/app/data/constants.js:42]: These variables shouldn't be deleted. They are used in the code

File: app/javascript/app/data/constants.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const { ESP_API } = process.env;
 export const LOGIN_URL = `${GFW_API}/auth/`;
 export const LOGIN_PARAMS = `?applications=climate-watch&token=true&callbackUrl=${location.origin}${CW_API}/auth/login`;
 export const LOGOUT_URL = '/auth/logout';
-export const ESP_HOST = ESP_API && ESP_API.replace('/api/v1', '');
```
Comment (Bluesmile82) [inline app/javascript/app/data/constants.js:15]: Is there a reason why you changed this?

File: app/javascript/app/data/constants.js [hunk 3]
```diff
@@ -301,7 +289,7 @@ export const GHG_TABLE_HEADER = {
 };
 
 export const WRI_WEBSITE = 'https://www.wri.org/';
-export const WRI_CLIMATE_BLOG = 'https://www.wri.org/blog-tags/14091';
+export const WRI_CLIMATE_BLOG = 'https://www.wri.org/resources/tags/14091';
```
Comment (Bluesmile82) [inline app/javascript/app/data/constants.js:292]: 👍

**Review by Bluesmile82 — CHANGES_REQUESTED** [review]

File: app/javascript/app/data/constants.js (hunk 1 above)
Comment (sw1115) [reply app/javascript/app/data/constants.js:42]: Thanks a lot for checking @Bluesmile82 ! Sorry just saw your comments here and yes I guess the only change needed is the "resources" in the URL

Comment (Bluesmile82) [reply app/javascript/app/data/constants.js:42]: No worries! I'll close this PR and make the change on a different one. Thanks!

**Comment by Bluesmile82** [comment]
@sw1115  I'm closing this PR. The changes have been applied already

**Pull request closed without merging**